tests/conftest.py

```diff
@@ -63,7 +63,7 @@ def python_conda_mirror_main_conda_forge_environment(tmpdir_factory):
 # conda-locks LockSpecification object
 @pytest.fixture(scope="function")
 def lock_spec_fixture(python_conda_mirror_main_conda_forge_environment) -> LockSpecification:
-    lock_spec = get_lock_spec_for_environment_file(python_conda_mirror_main_conda_forge_environment)
+    lock_spec = get_lock_spec_for_environment_file(python_conda_mirror_main_conda_forge_environment, "linux-64")
```

Comment:
The function only accepts one parameter, but "linux-64" is being passed too.

Reply:
This PR includes an update to require an additional parameter (due to conda-lock) changes. Here is the update in thie PR: https://github.com/MetroStar/conda-vendor/blob/04ff7907eb387fc12fc7ce6a46b4ccf6d41fe529/conda_vendor/conda_vendor.py#L19

Reply:
Oh, sorry.  I think I was looking at main by mistake.